### PR TITLE
revert to ruby 2.0 support

### PR DESCRIPTION
**Pull request by richm**
revert to ruby 2.0 support

### CODE REVIEW

**Comment by nhosoi** [comment]
/lgtm
How about the Dockerfile side in pr #786?
And this means we have to give up to upgrade to fluentd 1.0, right? :(

File: fluentd/Dockerfile.centos7 [hunk 1]
```diff
@@ -1,35 +1,39 @@
-FROM centos/ruby-24-centos7:latest
+FROM centos:centos7
 
 MAINTAINER OpenShift Development <dev@lists.openshift.redhat.com>
 
 ENV DATA_VERSION=1.6.0 \
-    FLUENTD_VERSION=0.12.41 \
-    FLUENTD_AUDIT_LOG_PARSER_VERSION=0.0.5 \
```
Comment (portante) [inline fluentd/Dockerfile.centos7]: Why do we also drop the `FLUENTD_AUDIT_LOG_PARSER_VERSION` variable?

File: fluentd/Dockerfile.centos7 [hunk 2]
```diff
@@ -1,35 +1,39 @@
...
+    FLUENTD_VERSION=0.12.42 \
```
Comment (portante) [inline fluentd/Dockerfile.centos7]: Can you also post a comment as to why we bump to `FLUENTD_VERSION` to `0.12.42`?

**Comment by richm** [comment]
> How about the Dockerfile side in pr #786?

I will submit another PR for that.

> And this means we have to give up to upgrade to fluentd 1.0, right? :(

Right, at least for the short term

File: fluentd/Dockerfile.centos7 (hunk 1 above)
Comment (richm) [reply fluentd/Dockerfile.centos7]: It is now unused

File: fluentd/Dockerfile.centos7 (hunk 2 above)
Comment (richm) [reply fluentd/Dockerfile.centos7]: It is the latest available 0.12.x version - I suppose I could have bumped that version in a separate commit

**Pull request merged**